greedy_algorithms/jumpgame.cpp

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file
+ * \brief Implementation of an algorithm to solve the jumping game problem
+ * \details
+ * This algorithm is a greedy algorithm.
+ * This solution takes in input as a vector and output as a boolean to check if you can reach the last position.
+ * @author [Rakshaa Viswanathan](https://github.com/rakshaa2000)
+*/
+
+#include <vector>
+#include<iostream>
+#include <cassert>
+
+//Implements the algorithm
+/*We name the indices good and bad based on whether we can reach the destination if we start at that position.
+ *We initialize the last index as lastPos.
+ *Here, we start from the end of the array and check if we can ever reach the first index.
+ *We check if the sum of the index and the maximum jump count given is greater than or equal to the lastPos.
+ *If yes, then that is the last position you can reach starting from the back.
+ *After the end of the loop, if we reach the lastPos as 0, then the destination can be reached from the start position.
```

Comment:
This shall be added in the first comment block, in `@details`.
See [`backtracking/n_queens.cpp`](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/backtracking/n_queens.cpp) as a reference. 🙂